.syncpackrc

```diff
@@ -52,7 +52,9 @@
 				"@wordpress/style-engine",
 				"@wordpress/stylelint-config",
 				"@wordpress/url",
-				"eslint*"
+				"eslint*",
+				"stylelint",
+				"stylelint-sass",
```

Comment:
`stylelint-sass` version is not pinned yet, so let's drop this line.

Reply:
Thanks, removed.